cmd/app.go

```diff
@@ -948,6 +960,27 @@ func parseExternalID(e string) (map[string]string, error) {
 	return parsed, nil
 }
 
+func getFilteredTemplate(ctx *cli.Context, c *cliclient.MasterClient, templateID string) (*managementClient.Template, error) {
```

Comment:
feels weird to call this getFilteredTemplate, you dont allow any filters and just pull and auto set the version. not critical tho